lib/ssl/ssl3con.c

```diff
@@ -2707,12 +2267,6 @@ ssl3_ComputeRecordMACConstantTime(
     PORT_Assert(inputLen >= spec->mac_size);
     PORT_Assert(originalLen >= inputLen);
 
-    if (spec->bypassCiphers) {
-        /* This function doesn't support PKCS#11 bypass. We fallback on the
-         * non-constant time version. */
-        goto fallback;
```

Comment:
If there is only one case that calls that code, move it inline and remove the goto.